sdk/go/auto/workspace.go

```diff
@@ -165,8 +169,18 @@ type ConfigValue struct {
 
 // ConfigOptions is a configuration option used by a Pulumi program.
 // Allows to use the path flag while getting/setting the configuration.
+// Allows to use the config file flag while getting/setting the configuration.
```

Comment:
I feel like these two comments are misplaced; not sure why. Let's move them above their respective fields.